Documentation/Artifacts.md

```diff
@@ -31,10 +31,11 @@ git "https://enterprise.local/desktop/git-error-translations2.git"
 
 ##### Binary only frameworks
 
-Dependencies that are only available as compiled binary `.framework`s are specified with the `binary` keyword and an https address that returns a binary project specification:
+Dependencies that are only available as compiled binary `.framework`s are specified with the `binary` keyword and as an https address, or a relative path, that returns a binary project specification:
```

Comment:
I would change this to:


Dependencies that are only available as compiled binary `.framework`s are specified with the `binary` keyword and as an `https://` URL, a `file://` URL, a relative path or an absolute path with no scheme, that returns a binary project specification